vars.tf

```diff
@@ -11,6 +11,12 @@ variable "aws_secret_key" {
   default = null
 }
 
+variable "app_env" {
+  description = "The abbreviated app environment (e.g. prod or stg)"
+  default     = "prod"
+  type        = string
+}
+
```

Comment:
This can be omitted as long as you change the "b2" remote as you suggested above.